src/particles/diagnostics/DiagnosticOutput.cpp

```diff
@@ -64,9 +64,9 @@ namespace impactx::diagnostics
                         amrex::ParticleReal const pt = part_pt[i];
 
                         // write particle data to file
-                        amrex::PrintToFile("output_beam.txt") << x << " " << y << " ";
-                        amrex::PrintToFile("output_beam.txt") << t << " " << px << " ";
-                        amrex::PrintToFile("output_beam.txt") << py << " " << pt << "\n";
+                        amrex::PrintToFile("diags/output_beam.txt") << x << " " << y << " ";
```

Comment:
It would be more efficient to merge the three calls to PrintToFile into one.

Reply:
es, that makes sense.  Line 67 of course needs to be continued to include t, px, py, and pt.